app/javascript/src/components/Common/SideMenuBar.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+import PropTypes from "prop-types";
+
+const SideMenuBar = ({ showSideMenuBar, title, menuData }) => {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(false);
+
+  return (
+    <MenuBar showMenu={showSideMenuBar} title={title}>
+      {menuData.filterTypes?.map(({ label, dataCount, active }, idx) => (
```

Comment:
Why's there an option chain over here?

Reply:
Yes. We don't need to add this check here.